examples/image_example.py

```diff
@@ -0,0 +1,71 @@
+"""
+Example that enables images from a camera and writes a single frame to a file.
+"""
+
+import argparse
+import asyncio
+import logging
+import os
+import qtm_rt
+import xml.etree.ElementTree as ET
+
+
+def output_format(output_path):
+    extension = os.path.splitext(output_path)[1].lower()
+    if extension == ".jpg":
+        return "jpg"
+    if extension == ".png":
+        return "png"
+
+    raise RuntimeError("Unsupported output format {extension}")
+
+
+def enable_disable_cameras(output_path, target_camera_id, xml_str):
+    format = output_format(output_path)
+    xml = ET.fromstring(xml_str)
+    for camera in xml.findall("./Image/Camera"):
+        camera_id = camera.find("ID").text
+        is_target_camera = camera_id == target_camera_id
+        logging.info("Setting Enabled for Camera %s to %s", camera_id, is_target_camera)
+        camera.find("Enabled").text = str(is_target_camera)
+
+        if is_target_camera:
+            camera.find("Format").text = format
```

Comment:
Some might consider this a stupid question, but I want to ask it regardless: **Can more than one camera have the same ID?** Purely based on what "ID" means in my world view, I would assume each camera has a unique ID. If this is the case, do we want to exit the for-loop early? To make the code a little more explicit?

Reply:
No, the IDs are unique. They are basically the camera numbers as QTM counts/numbers the cameras.